src/common/Tiles/PeopleTile/styled.js

```diff
@@ -3,56 +3,70 @@ import { ReactComponent as MissingPhotoIcon } from "../../../images/missingProfi
 import { boxShadowColor } from "../../styled";
 
 export const PeopleTileContainer = styled.div`
-    background-color: ${({ theme }) => theme.color.white};
-    width: 100%;
-    max-width: 208px;
-    max-height: auto;
-    display: grid;
-    grid-template-rows: auto, repeat(2, 1fr);
-    padding: 16px;
-    border-radius: 5px;
-    box-shadow: 0px 4px 12px 0px ${boxShadowColor};
+  background-color: ${({ theme }) => theme.color.white};
+  width: 100%;
+  max-width: 208px;
+  height: 339px;
```

Comment:
I think it would be better to set **height: auto;**

Reply:
Maybe you didn't pull all the changes? It displays correctly on my end.